File: app/src/main/java/com/intelegain/agora/Controller/VerticalSpaceItemDecoration.java
package com.intelegain.agora.Controller;

import android.graphics.Rect;
import android.view.View;

import androidx.recyclerview.widget.RecyclerView;

/**
 * Created by Admin on 22/08/16.
 */
public class VerticalSpaceItemDecoration  extends RecyclerView.ItemDecoration
{

    private final int mVerticalSpaceHeight;

    public VerticalSpaceItemDecoration(int mVerticalSpaceHeight) {
        this.mVerticalSpaceHeight = mVerticalSpaceHeight;
    }

    @Override
    public void getItemOffsets(Rect outRect, View view, RecyclerView parent,
                               RecyclerView.State state) {
        outRect.bottom = mVerticalSpaceHeight;
    }
}

File: app/src/main/java/com/intelegain/agora/Controller/VerticalSpaceItemDecoration.kt
package com.intelegain.agora.Controller

import android.graphics.Rect
import android.view.View
import androidx.recyclerview.widget.RecyclerView
import androidx.recyclerview.widget.RecyclerView.ItemDecoration

/**
 * Created by Admin on 22/08/16.
 */
class VerticalSpaceItemDecoration(private val mVerticalSpaceHeight: Int) : ItemDecoration() {
    override fun getItemOffsets(outRect: Rect, view: View, parent: RecyclerView,
                                state: RecyclerView.State) {
        outRect.bottom = mVerticalSpaceHeight
    }

}